proto/proto-backplane-grpc/build.gradle

```diff
@@ -38,7 +38,7 @@ dependencies {
   // technically we should have a runtime dependency on barrage-core, but instead we are regenerating
   // the same output that it contains, and we have declared the correct dependencies as necessary
   //  compile 'io.deephaven.barrage:barrage-core:0.4.0'
```

Comment:
Probably a silly request, but can you bump the barrage version in the comment as well?